alchemy/tests/test_alchemy.py

```diff
@@ -302,6 +307,137 @@ def test_annihilated_states(platform_name=None, precision=None):
+        del context, integrator
+        return potential
+
+    # Compute energy component decomposition of standard system NonbondedForce
+    system = copy.deepcopy(factory.reference_system)
+    # Sterics (nonbonded plus exceptions)
+    forces_to_remove = list()
+    # Remove all forces but NonbondedForce
+    for (force_index, force) in enumerate(system.getForces()):
+        if force.__class__.__name__ != 'NonbondedForce':
+            forces_to_remove.append(force_index)
+    for force_index in reversed(forces_to_remove):
+        system.removeForce(force_index)
+    # Compute sterics + exceptions
+    PSE_ESE = compute_energy(system, positions)
+    # Turn off all charges, leaving all exceptions
+    for (force_index, force) in enumerate(system.getForces()):
+        if force.__class__.__name__ == 'NonbondedForce':
+            for particle_index in range(system.getNumParticles()):
+                [charge, sigma, epsilon] = force.getParticleParameters(particle_index)
+                force.setParticleParameters(particle_index, abs(0*charge), sigma, epsilon)
+    # Compute sterics + exceptions
+    PS_ESE = compute_energy(system, positions)
+    # Turn off all direct interactions
+    for (force_index, force) in enumerate(system.getForces()):
+        if force.__class__.__name__ == 'NonbondedForce':
+            for particle_index in range(system.getNumParticles()):
+                [charge, sigma, epsilon] = force.getParticleParameters(particle_index)
+                force.setParticleParameters(particle_index, abs(0*charge), sigma, abs(0*epsilon))
+    # Compute exception energy only
+    ESE = compute_energy(system, positions)
+    # Turn off all electrostatics exceptions
+    for (force_index, force) in enumerate(system.getForces()):
+        if force.__class__.__name__ == 'NonbondedForce':
+            for exception_index in range(force.getNumExceptions()):
+                [iatom, jatom, chargeprod, sigma, epsilon] = force.getExceptionParameters(exception_index)
+                force.setExceptionParameters(exception_index, iatom, jatom, abs(0*chargeprod), sigma, epsilon)
+    # Compute sterics + exceptions
+    ES = compute_energy(system, positions)
+    # Turn off all sterics exceptions
+    for (force_index, force) in enumerate(system.getForces()):
+        if force.__class__.__name__ == 'NonbondedForce':
+            for exception_index in range(force.getNumExceptions()):
+                [iatom, jatom, chargeprod, sigma, epsilon] = force.getExceptionParameters(exception_index)
+                force.setExceptionParameters(exception_index, iatom, jatom, abs(0*chargeprod), sigma, abs(0*epsilon))
+    # Energy should be zero.
+    off = compute_energy(system, positions)
+    assert (off == 0.0), "Energy should be zero when electrostatics and sterics particle and exceptions are turned off."
+
+    # Compute other values.
+    PS = PS_ESE - P_ESE
```

Comment:
This is the line that is making tests fail right now. I think you meant `PS = PS_ESE - ESE`?